server/src/com/cloud/storage/LocalStoragePoolListener.java

```diff
@@ -39,7 +39,7 @@
 import com.cloud.utils.db.DB;
 
 public class LocalStoragePoolListener implements Listener {
-    private final static Logger s_logger = Logger.getLogger(LocalStoragePoolListener.class);
+    // private final static Logger s_logger = Logger.getLogger(LocalStoragePoolListener.class);
```

Comment:
While making changes in this file, I noticed that Eclipse was pointing out that the variable s_logger was not in use. Instead of removing the entire line, I just commented it out in case someone later wanted to easily resume making use of this variable.